app/views/robots/show.text.erb

```diff
@@ -0,0 +1,13 @@
+<%- if Rails.env.production? %>
```

Comment:
This needs to be `Rails.configuration.app_role.production?` (`Rails.env` will always be production for deployed apps)